test/Replay/libkleeruntest/replay_cex_incorrect_result.c

```diff
@@ -0,0 +1,50 @@
+// RUN: %clang %s -S -emit-llvm -g -c -o %t.ll
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out %t.ll
```

Comment:
Can you add a deterministic searcher? Otherwise, the generated tests might be in different order and therefore fail.
```suggestion
// RUN: %klee --search=dfs --output-dir=%t.klee-out %t.ll
```

Reply:
I make new test in the same way as in 
https://github.com/klee/klee/blob/master/test/Replay/libkleeruntest/replay_klee_prefer_cex.c
There is no --dfs flag in old test. And this flag broke the test, I will fix it